include/z64.h

```diff
@@ -42,13 +42,28 @@
 #define REGION_JP 2
 #define REGION_EU 3
 
-#define Z_PRIORITY_MAIN        10
-#define Z_PRIORITY_GRAPH       11
-#define Z_PRIORITY_AUDIOMGR    12
-#define Z_PRIORITY_PADMGR      14
-#define Z_PRIORITY_SCHED       15
-#define Z_PRIORITY_DMAMGR      16
-#define Z_PRIORITY_IRQMGR      17
+#define THREAD_PRI_IDLE_INIT    10
```

Comment:
The idea behind the Z_PRIORITY names was to follow the same scheme as thread.h defines while having Z_ instead of OS_ to differentiate them as being custom for the game(s).

I guess the intent here is to move away from that and the names seem good, but I find it a bit odd to have OS_PRIORITY_ and THREAD_PRI_ for the same group of values, so I'm not sure if it's a good idea or if there is a better solution

Reply:
It does seem a bit weird to me too at first now that you pointed it out, but on second thought we also don't have z_ prefixes for every single thing that isn't libultra so it's probably fine to not have any here as well

the os_priority_ defines in libultra also lack the keyword "thread", which we can't add because original names, it's fine in the context of libultra but "z_priority_" could be easily confused with other priorities in "zelda" context imo

all in all I think "thread" should stay in the name, so if we still wanted to make `THREAD_PRI_` look more like `OS_PRIORITY_`, it would be maybe `Z_THREAD_PRIORITY_` (?) which huh why not but also doesn't seem needed?